gemrb/core/Game.cpp

```diff
@@ -1589,13 +1595,6 @@ void Game::UpdateScripts()
 		}
 	}
 
-	// perhaps a StartMusic action stopped the area music?
-	// (we should probably find a less silly way to handle this,
-	// because nothing can ever stop area music now..)
-	if (!core->GetMusicMgr()->IsPlaying()) {
-		ChangeSong(false,false);
-	}
-
```

Comment:
I'm not sure this is really equivalent. What if the previous ChangeSong call caused a failure for whatever reason?

Reply:
Indeed this isn't fully equivalent. But does this really serve a purpose? When the ChangeSong failed in the previous call why should it succeed just a moment later? The internal parameters won't have changed then.